depends/builders/darwin.mk

```diff
@@ -1,5 +1,5 @@
-build_darwin_CC:=$(shell xcrun -f clang)
-build_darwin_CXX:=$(shell xcrun -f clang++)
+build_darwin_CC:=clang
+build_darwin_CXX:=clang++
```

Comment:
Can you clarify why this is needed? I did look at the commit comments about passing `--sysroot` and missing includes, however from what I can tell, there should be no change in behaviour with this diff. I did run a few compilation tests and didn't see any obvious difference in compile flags/includes etc. 

My assumption is that the problems seen (https://github.com/chaincodelabs/libmultiprocess/issues/5, https://github.com/bitcoin/bitcoin/pull/16367#issuecomment-592093081) might have been due to an issue with the development environments i.e missing SDK headers, not installing the CommandLineTools etc.

Mostly curious because if this was an issue it seems like it shouldn't be multiprocess specific? It also seems strange to drop the `xcrun` call only for clang/clang++, and not for the rest of the tools.